src/components/articles/articleHeader/ArticleHeader.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { MDBContainer, MDBRow, MDBCol } from "mdbreact";
+import "./ArticleHeader.scss";
+import ArticleOptions from "../articleOptions/ArticleOptions";
+
+const ArticleHeader = props => {
+  const { article, canModify, handleDelete, handleLink } = props;
+  return (
+    <MDBContainer className="mt-5 ">
+      <MDBRow>
```

Comment:
You have both the ArticleHeader.jsx and NavBar.jsx which is a duplication of work.